uber/templates/jobs/index.html

```diff
@@ -68,20 +68,20 @@
         </div>
 
 </div>
-{% comment %}
+{# comment
```

Comment:
Don't know what's happening here, but it seemed safer to leave it in. I'd prefer removing it altogether if there's  not a good reason to keep it.